backend/package.json

```diff
@@ -8,6 +8,7 @@
   "main": "src/app.js",
   "scripts": {
     "ci": "npm run lint && npm test",
+    "kinesis": "node bin/kinesis.js",
```

Comment:
I can't see where this task is being called from. Is this prep for more to come? If it supports local dev / test can we include an entry in the readme? Possibly rename to local-kinesis.